spec/index.html

```diff
@@ -459,11 +460,16 @@ <h2>Simple Interpretations</h2>
       </tr>
 
       <tr>
-        <td class="semantictable">if E is an IRI then I(E) = IS(E)</td>
+        <td class="semantictable">if E is an IRI, then I(E) = IS(E)</td>
       </tr>
 
       <tr>
-        <td class="semantictable">if E is a ground triple `s p o.`
+        <td class="semantictable">if E is a triple term, then I(E) = RE(I(E.s), I(E.p), I(E.o)),<br/>
```

Comment:
```suggestion
        <td class="semantictable">if E is a ground triple term, then I(E) = RE(I(E.s), I(E.p), I(E.o)),<br/>
```
While this should be clear from the context (the caption of the table says this is about ground graph), since the entries about (asserted) triples and graph restate that they are ground, this should also be the case here.